libraries/AP_GyroFFT/CMSIS_5/include/arm_math.h

```diff
@@ -381,7 +381,7 @@ extern "C"
     ARM_MATH_NANINF         = -4,        /**< Not-a-number (NaN) or infinity is generated */
     ARM_MATH_SINGULAR       = -5,        /**< Input matrix is singular and cannot be inverted */
     ARM_MATH_TEST_FAILURE   = -6         /**< Test Failed */
-  } arm_status;
+  } arming_status;
```

Comment:
Just to be clear in case you don't know `arm` here refers to the ARM architecture of the computer chip.

I'm not sure the best name for this... but maybe `arm_arch_math_status` or `arm_math_status`?

Reply:
armcpu? (it's definitely confusing right now)

Reply:
`cmsis_arm_status`

... namespaced like it should've been to start off with :-)